amundsen_application/static/config/config-default.ts

```diff
@@ -22,6 +22,7 @@ const configDefault: AppConfig = {
     },
     iconPath: 'PATH_TO_ICON'
   },
+  logoPath: null,
```

Comment:
It would be good if we could comment about the configs for external users(e.g example value, what it is used for)?

Reply:
We currently mention it in our README: https://github.com/lyft/amundsenfrontendlibrary#application-config, and we also put some lengthier descriptions in this file for the less straightforward cases: https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/config/config.types.ts